src/transforms/uniquecoords.jl

```diff
@@ -52,39 +51,30 @@ function apply(transform::UniqueCoords, data::Data)
   # group locations with the same coordinates
   pts = [centroid(dom, i) for i in 1:nelements(dom)]
   X = reduce(hcat, coordinates.(pts))
-  ginds = _uniqueinds(X, 2)
-  groups = Dict(ind => Int[] for ind in unique(ginds))
-  for (i, ind) in enumerate(ginds)
-    push!(groups[ind], i)
-  end
+  uinds = _uniqueinds(X, 2)
+  ginds = unique(uinds)
+  groups = [findall(==(ind), uinds) for ind in ginds]
```

Comment:
I couldn't think of a more efficient way to collect group indices. Do you have any suggestions @juliohm?

Reply:
Resolved!